applications/nrf5340_audio/src/bluetooth/bt_content_control/content/bt_media_control.c

```diff
@@ -184,101 +236,172 @@ static void mcs_local_player_instance_cb(struct media_player *player, int err)
 	(void)media_proxy_ctrl_send_command(local_player, &cmd);
 }
 
-int ble_mcs_discover(struct bt_conn *conn)
+/**
+ * @brief	Send play or pause command to either local media player or peer
+ *
+ * param	cmd	Command to send
+ *
+ * @return	0 for success, error otherwise.
+ */
+static int play_pause(struct mpl_cmd *cmd)
+{
+	int ret = 0;
+
+	if (IS_ENABLED(CONFIG_BT_MCS)) {
+		ret = media_proxy_ctrl_send_command(local_player, cmd);
+	} else if (IS_ENABLED(CONFIG_BT_MCC)) {
+		/* Send cmd to all peers connected and has finished discovery */
+		for (uint8_t i = 0; i < CONFIG_BT_MAX_CONN; i++) {
```

Comment:
```suggestion
		for (int i = 0; i < ARRAY_SIZE(mcc_peer); i++) {
```

Reply:
Will change to ARRAY_SIZE, but not int. No need for either 32 bits or negative values

Reply:
We mostly use "int" in for-loops in our application.
We should do it the same way in the whole application